paint/PAINT_StatusBarWindow.h

```diff
@@ -9,7 +9,7 @@ namespace paint
 	{
 	public:
 		StatusBarWindow() = delete;
-		StatusBarWindow(gfx::Renderer* renderer, gfx::Rectangle const& rect, const char* name, const int drawWinX, const int drawWinY);
+		StatusBarWindow(gfx::Renderer* renderer, gfx::Rectangle const& rect, const char* name, int drawWinX, int drawWinY);
```

Comment:
Don't construct with the renderer, just pass it into draw. Then there's no scope for doing renderer-related things in non-renderer-related contexts.
**MUST**

Reply:
removed renderer_ pointer for windows where it's not necessary, and added draw overload with renderer